r-package/grf/vignettes/rate.Rmd

```diff
@@ -0,0 +1,216 @@
+
+Another thing to note is that the "high-vs-low" out-of-bag CATE quantile construction used in Athey and Wager (2019) can also be seen as a kind of RATE, where in place of Qini's identity weighting it would have point mass at q = a given quantile.
+
+## Estimating the RATE
+The overview in the previous section gave a stylized introduction where we imagined we knew $\tau(X_i)$. In practice these have to be estimated and the appropriate *feasible definition* (using $S(X_i) = \hat \tau(X_i)$ as an example) is: the TOC curve ranks all observations on a test set $X^{test}$ according to a CATE function $\hat \tau^{train}(\cdot)$ estimated on a training set, and compares the ATE for the top q-th fraction of units prioritized by $\hat \tau^{train}(X^{test})$ to the overall ATE:
+
+\begin{equation*}
+\begin{split}
+&\textrm{TOC}(q) = E[Y^{test}_i(1) - Y^{test}_i(0) \,|\, \hat \tau^{train}(X^{test}_i) \geq \hat F^{-1}(1 - q)]\\
+& \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \  - E[Y^{test}_i(1) - Y^{test}_i(0)],
+\end{split}
+\end{equation*}
+
+where $\hat F$ is the empirical distribution function of $\hat \tau^{train}(X^{test})$. GRF estimates
+
+$$\tau(X_i)= [Y^{test}_i(1) - Y^{test}_i(0) \,|\, X_i = x]$$
+
+using a suitable doubly robust score construction $\Gamma_i \approx \tau(X_i)$ which is valid in a variety of settings, enabling inference with the bootstrap, and the final AUTOC estimate is constructed from $\frac{1}{n} \sum_{i=1}^{n} \textrm{TOC}(\frac{i}{n})$. This functionality is implemented in the function `rank_average_treatment_effect` which as first argument takes an "evaluation" forest which computes the appropriate score construction $\hat \Gamma_i$ and second argument estimated prioritization scores for units in the evaluation set, using for example a CATE function estimated on a training set (with additional support for forests trained with sample weights or clusters).
+
+## An application to SPRINT and ACCORD
+To illustrate RATE we consider an example application from a medical setting. Two large randomized trials *ACCORD* (ACCORD Study Group, 2010) and *SPRINT* (SPRINT Research Group, 2015) conducted on similar populations and designed to measure the effectiveness of a hypertension treatment find that the treatment is effective. However, the two trials differs in the magnitude of the estimated benefit. Various explanations for this finding have been proposed, we'll focus on one in particular here: the hypothesis that the difference is due to *heterogeneity in treatment effects* (see Yadlowsky et al., 2021 for references).
```

Comment:
To clarify, the two trials actually came to different conclusions. *SPRINT* found the treatment was effective, *ACCORD* found that the treatment was not effective.

More specifically,

*[SPRINT](https://www.nejm.org/doi/full/10.1056/nejmoa1511939)* found that "Among patients at high risk for cardiovascular events but without diabetes, targeting a systolic blood pressure of less than 120 mm Hg, as compared with less than 140 mm Hg, resulted in lower rates of fatal and nonfatal major cardiovascular events and death from any cause". 

On the other hand, *[ACCORD](https://www.nejm.org/doi/full/10.1056/nejmoa1001286)* found that "In patients with type 2 diabetes at high risk for cardiovascular events, targeting a systolic blood pressure of less than 120 mm Hg, as compared with less than 140 mm Hg, did not reduce the rate of a composite outcome of fatal and nonfatal major cardiovascular events."

Reply:
Can you use github's "add a suggestion" when making comments so I can just commit your updated paragraph directly?

And is it possible to rephrase it using less (none if possible) medical jargon?  A newbie from a social science background will only understand a fraction of those words (composite included). Remember the way the example is represented here is supposed to demonstrate RATE for a newbie, I think it's safe to gloss over all the finer details that are not necessary to understands that you can use RATE in this setting (all those details are in the arxiv anyways)

Reply:
Yes! Sure thing - thanks for the "add a suggestion" tip, I'll definitely use that going forward! And totally agreed re:limiting jargon. Sorry, that was confusing - I'm not at all suggesting that we include unnecessary details about the trials, just wanted to clarify what the actual results were so we're all on the same page.